src/main/java/io/antmedia/console/rest/JWTServerFilter.java

```diff
@@ -0,0 +1,86 @@
+package io.antmedia.console.rest;
+
+import java.io.IOException;
+import java.security.interfaces.RSAPublicKey;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import io.antmedia.settings.ServerSettings;
+
+
+import com.auth0.jwk.Jwk;
+import com.auth0.jwk.JwkException;
+import com.auth0.jwk.JwkProvider;
+import com.auth0.jwk.UrlJwkProvider;
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import io.antmedia.filter.AbstractFilter;
+
+
+public class JWTServerFilter extends AbstractFilter {
+
+	private ServerSettings serverSettings;
+	public static final String JWT_TOKEN = "Authorization";
+
+
+	@Override
+	public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+			throws IOException, ServletException {
+		HttpServletRequest httpjwtRequest =(HttpServletRequest)request;
+		String path = ((HttpServletRequest) request).getRequestURI();
+		serverSettings = getServerSetting();
+
+		if(serverSettings != null && !serverSettings.isJwtServerControlEnabled() || (httpjwtRequest.getHeader(JWT_TOKEN) != null && checkJWT(httpjwtRequest.getHeader(JWT_TOKEN)))) {
+			chain.doFilter(request, response);
+			logger.info("Request {} is completed",path);
+		}
+		else {
+			HttpServletResponse resp = (HttpServletResponse) response;
+			resp.setStatus(HttpServletResponse.SC_FORBIDDEN);
+		}
+	}
+
+	private boolean checkJWT( String jwtString) {
+		boolean result = true;
```

Comment:
Better approach is making the default value for `result` is `false` and making it `true` if it's successful. Because it returns wrong values for the unexpected cases.